.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+## Issue: <!-- link the issue or issues this PR resolves here -->
+<!-- If your PR depends on changes from another pr link them here and describe why they are needed on your solution section. -->
```

Comment:
Nit: `...from another PR link them here and describe why in your solution`

Reply:
Updated!